QUARKUS-5666.md

```diff
@@ -0,0 +1,43 @@
+# QUARKUS-5666 Add support for encrypted PKCS#8 - technology preview
+
+### Links to the resources
+
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5666
+- GH PR related to the JIRA issue: https://github.com/quarkusio/quarkus/pull/44549
+- Guide modified and related:
+    * https://quarkus.io/guides/tls-registry-reference
+
+## Scope of the testing
+Modification of the Test Framework (TF) to introduce Format.ENCRYPTED_PEM in the Certificate class to use it with the @Certificate 
```

Comment:
I don't know man, the _scope of the testing_ should tell me what to test, not what how you will facilite testing. Information in this section are valid and important, but they belong to the _Impact on test suites and testing automation_ section or to your custom _Changes in the Test Framework_ section.